features/netsocket/NetworkStack.h

```diff
@@ -184,6 +184,7 @@ class NetworkStack: public DNS {
 protected:
     friend class InternetSocket;
     friend class UDPSocket;
+    friend class RAWSocket;
```

Comment:
`RawIPSocket`. RAW is not an acronym, and I think we need to distinguish from non-IP cases.

Reply:
If there some way to create an ICMP Socket by re-using UDPSocket class ( such as InternetDatagram Socket with ICMP protocol it would be nice)